### PR TITLE
Chocolatey servers update

### PR DESCRIPTION
**Pull request by terence-octo**
# About

This is a post about rewriting the Chocolatey servers post for Q2 2022

# Release

To be merged once ready, this will delete the old post

# Blog image idea

The design team will work on a design for the blog image. Add any ideas you have about the graphic image here.

# Pre-requisites

- [ ] The draft is complete and this post is ready to be reviewed.
- [ ] I'm familiar with the [writing for Octopus TL;DR](https://www.octopus.design/writing/writing-tldr/).
- [ ] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [ ] The images I've included follow the [rules for working with images](https://www.octopus.design/writing/working-with-images/).
- [ ] I've added alt text for all the images I've included (alt text describes image to people unable to see it, 125 characters max)
- [ ] One screenshot (if any are included) is suitable for Twitter. This screenshot should be 16:9 ratio and at least 1200 x 675 pixels. For screenshots from the Octopus Web Portal, please use Chrome and the browser’s zoom function to remove as much white space as possible. If screenshots are annotated (e.g., with boxes or arrows), please also provide a version that is unannotated for use on Twitter. Do not use dark mode for the Octopus Web Portal.

### CODE REVIEW

**Review by TeganAli — CHANGES_REQUESTED** [review]

File: blog/2022-q2/chocolatey-powershell-and-runbooks/index.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+When installing SQL, create your own Chocolatey Package. SQL is tricker to install as it requires an administrator account to install quickly, and you want to configure things such as default users, groups, and locations of the database and log files.
+
+When using Octopus to install SQL Developer Edition or SQL Express, you can do it without the Tentacle running as a local administrator. You still need to use the optional location for the files. Another gotcha is that if you run the install under a service account with a named service account, by default, it uses that user as the default SQL administrator. You need to connect with that account to give yourself access.
+:::
+
+## Upgrading Chocolatey packages
+
+You can use Octopus to install applications via Chocolatey, but software gets patched, and new features and security enhancements are added all  the time. What happens when you need the latest version of the software you installed using runbooks and Chocolatey?
+
+You [upgrade](https://chocolatey.org/docs/commandsupgrade) it with Chocolatey and runbooks. This is easy to do, and one of my favorite commands on logging in to a server, or for running using a runbook is:
```
[the remark this answers is not in the document]
Comment (terence-octo) [reply blog/2022-q2/chocolatey-powershell-and-runbooks/index.md]: I have updated this to shorten the main point, hope it works!

File: blog/2022-q2/chocolatey-powershell-and-runbooks/index.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+- **Package Source (Optional)**: This is probably the *single most crucial parameter in this step*. If you’re doing this at home, it might be acceptable to use the Chocolatey Package Repository, which is the default setting. However, if you’re doing this for a company, please consider using your own package source repositories, such as [Nexus](https://www.sonatype.com/nexus/repository-pro), [Artifactory](https://jfrog.com/artifactory/), or [MyGet](https://www.myget.org/).
+
+The Chocolatey Package resource is built by the community for the community. If you’re using the community repository for enterprise or large scale package installation, you will likely be [rate limited](https://chocolatey.org/docs/community-packages-disclaimer#rate-limiting). Be careful, and be kind to the community.
+
+The last two options cover whether you want to see the download progress in your logs. I usually turn them off to avoid thousands of log files. The other is to allow for additional parameters:
```
[the remark this answers is not in the document]
Comment (terence-octo) [reply blog/2022-q2/chocolatey-powershell-and-runbooks/index.md]: I have updated this to make it clear which option is being referred to

[the remark this answers is not in the document]
Comment (TeganAli) [reply blog/2022-q2/chocolatey-powershell-and-runbooks/index.md]: Perfect, thanks so much, Terence.

File: blog/2022-q2/chocolatey-powershell-and-runbooks/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (TeganAli) [reply blog/2022-q2/chocolatey-powershell-and-runbooks/index.md]: Awesome, thank you!

**Review by TeganAli — APPROVED** [review]

File: blog/2022-q2/chocolatey-powershell-and-runbooks/index.md [hunk 3]
```diff
@@ -0,0 +1,321 @@
+---
+title: Configuring Windows servers with Chocolatey, PowerShell, and Octopus Runbooks
+description: This blog post shows you how to automate your Windows server setup with Chocolatey, PowerShell, and Octopus Runbooks.
```
Comment (TeganAli) [inline blog/2022-q2/chocolatey-powershell-and-runbooks/index.md:3]:
```suggestion
description: This post shows you how to automate your Windows server setup with Chocolatey, PowerShell, and Octopus Runbooks.
```

**Pull request merged**